server/route.go

```diff
@@ -1506,6 +1510,10 @@ func (s *Server) routeAcceptLoop(ch chan struct{}) {
 	s.done <- true
 }
 
+func (s *Server) warnOfClusterTLSInsecure() {
+	s.Warnf("Server will skip verification of routes' certificate chain and hostname")
```

Comment:
Welcome any recommendation for the warning text :-)

Reply:
"TLS Hostname verification disabled, system will not verify identity of the solicited route"?

Reply:
Great! I think I will use the approach of a constant for this message so I don't need the function.